doozer/doozerlib/distgit.py

```diff
@@ -411,6 +412,13 @@ async def push_async(self):
         timeout = str(self.runtime.global_opts['rhpkg_push_timeout'])
         await exectools.cmd_assert_async(["timeout", f"{timeout}", "git", "push", "--follow-tags"], cwd=self.distgit_dir, retries=3)
 
+    def get_branch_el(self) -> Optional[int]:
```

Comment:
This replaced `isolate_rhel_major_from_version()` that is no longer used and can be removed

Reply:
Removed the import.

Reply:
I think `isolate_rhel_major_from_version()` can be wiped out, nothing is using it as this PR is using `isolate_el_version_in_brew_tag()` instead